.ci.yaml

```diff
@@ -3220,9 +3220,6 @@ targets:
       - .ci.yaml
 
   - name: Mac_x64 verify_binaries_codesigned
-    bringup: true
-    enabled_branches:
-      - flutter-\d+\.\d+-candidate\.\d+
```

Comment:
This won't be checked in, I want to see what happens in this PR running in presubmit.

Reply:
Good, both got all the way to the "expected" failure per https://github.com/flutter/flutter/issues/116611:
```
16:24:07.853741 exec-cmd.c:139          trace: resolved executable path from Darwin stack: /Volumes/Work/s/w/ir/cipd_bin_packages/bin/git
16:24:07.854131 exec-cmd.c:238          trace: resolved executable dir: /Volumes/Work/s/w/ir/cipd_bin_packages/bin
16:24:07.854567 git.c:460               trace: built-in: git checkout d9a5ea970fd4ed10f8480b690c91afb459186ad5
fatal: reference is not a tree: d9a5ea970fd4ed10f8480b690c91afb459186ad5
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8789864159913761313/+/u/Verify_x64_binaries_codesigned/verify_binaries_codesigned/test_stderr